spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDeploymentController.java

```diff
@@ -142,7 +145,19 @@ public ResponseEntity<String> manifest(@PathVariable("name") String name,
 	@RequestMapping(path = "/history/{name}", method = RequestMethod.GET)
 	@ResponseStatus(HttpStatus.OK)
 	public Collection<Release> history(@PathVariable("name") String releaseName) {
-		return this.streamService.history(releaseName);
+		ArgumentSanitizer sanitizer = new ArgumentSanitizer();
+		return this.streamService.history(releaseName)
+			.stream()
+			.map(release -> {
```

Comment:
Moving lines 151-156 to a method and then using a method reference here instead would make this a bit more readable. **[NIT]** though.

Reply:
Ok